discovery_engine/lib/src/infrastructure/assets/file_asset_fetcher.dart

```diff
@@ -0,0 +1,44 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import 'dart:io' show File;
+import 'dart:typed_data' show Uint8List;
+
+import 'package:xayn_discovery_engine/src/domain/assets/asset_fetcher.dart'
+    show AssetFetcher;
+import 'package:xayn_discovery_engine/src/domain/assets/data_provider.dart'
+    show DataProvider;
+import 'package:xayn_discovery_engine/src/logger.dart' show logger;
+
+class FileAssetFetcher extends AssetFetcher {
```

Comment:
So far this is implementation is not used anywhere. Let me know if we want to keep it, or should we remove it.

Reply:
If we don't use it then we can delete it, we can always add it back if we end up needing it.

(It's probably somewhere committed outside of this branch, which will be auto-deleted on merge?
Though then it's also easy to recreate I guess.)

Reply:
sure, removed